settings.gradle

```diff
@@ -2,6 +2,8 @@ rootProject.name = "gradle-baseline"
 enableFeaturePreview("STABLE_PUBLISHING")
 
 include "baseline-error-prone"
+include "baseline-refaster-plugin"
```

Comment:
I'd like to rename this to `baseline-refaster-javac-plugin` so it's clear this is not a gradlep plugin but an actual _compiler_ plugin!